MyBlog.Infrastructure/Entities/Article.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace MyBlog.Infrastructure.Entities;
+
+public class Article
+{
+    public int Id { get; set; }
+
+    public int UserId { get; set; }
+
+    public string Name { get; set; }
```

Comment:
Maybe, rename Name field to Title ?
I think it`ll be correct name for this property